packages/core/src/configuration.ts

```diff
@@ -170,11 +173,30 @@ const axisChart: AxisChartOptions = Tools.merge({}, chart, {
 	ruler,
 	zoomBar: {
 		zoomRatio: 0.4,
+		minZoomRatio: 0.01,
 		top: {
 			enabled: false,
 			type: ZoomBarTypes.GRAPH_VIEW
 		}
-	} as ZoomBarsOptions
+	} as ZoomBarsOptions,
+	toolbar: {
+		enabled: false,
+		maxIcons: 3,
+		controlsInOrder: [
+			{
+				type: ToolbarControlTypes.ZOOM_IN,
+				text: "Zoom in"
+			} as ToolbarControl,
+			{
+				type: ToolbarControlTypes.ZOOM_OUT,
+				text: "Zoom out"
+			} as ToolbarControl,
+			{
+				type: ToolbarControlTypes.RESET_ZOOM,
+				text: "Reset zoom"
+			} as ToolbarControl
+		]
```

Comment:
does this default to anything?
What if someone doesn't provide a `text` value for `ToolbarControlTypes.ZOOM_IN`? would it default to `Zoom in`?

Reply:
No, I expected user to provide the required `text`.
But in latest https://github.com/carbon-design-system/carbon-charts/pull/791/commits/ad3c9b0b6f6955f93c3daac07210106060739be9, I check the `text`, and use `type` if `text` is not available.

BTW, I also remove the default ToolbarControls in configuration because if will provide extra control setting if user provides less control configuration. (We use `Tools.merge` in `model.setOptions`)